ros/src/computing/perception/semantics/packages/costmap_generator/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ find_package(catkin REQUIRED COMPONENTS
   roscpp
   pcl_ros
   grid_map_core
```

Comment:
maybe we can remove this one? it is a dependency of `grid_map_ros`.

Reply:
@amc-nu thanks for pointing that out. I have removed it